datahub-web-react/src/app/entity/dataset/profile/schema/utils/schemaTitleRenderer.tsx

```diff
@@ -54,7 +56,9 @@ export default function useSchemaTitleRenderer(
         return (
             <>
                 <FieldPathContainer>
-                    <FieldPathText>{pathToDisplay}</FieldPathText>
+                    <FieldPathText>
+                        <Highlight search={filterText}>{pathToDisplay}</Highlight>
```

Comment:
fancy! bummer that it doesn't work on nested components though :/ i think we'll need to build that for our search cards otherwise we'll need to render like a zillion of these.